src/Messaging/src/Base/Support/MessageBuilder.save

```diff
@@ -0,0 +1,310 @@
+﻿// Copyright 2017 the original author or authors.
```

Comment:
Why do we need to keep this version of MessageBuilder in source?

Reply:
We don't ... deleted